dir_verify.py

```diff
@@ -30,17 +33,24 @@ def dir_verify(ch, method, properties, body):
 
             else:
                 if not path.exists():
-                    # Make sure folder exists and with right permission
-                    path.mkdir(mode=0o700, parents=True, exist_ok=True)
-
-                    # Make sure ownership is correct
-                    shutil.chown(path, int(msg['uid']), int(msg['gid']))
-
-                    logger.debug(f'{path} created')
-
-        msg['success'] = True
+                    # check if dirs exist and record any missing dirs
+                    missing_dirs.append(path)
+                    msg['success'] = False
+                    msg['errmsg'] = f"Error: missing dirs {missing_dirs}"
```

Comment:
use `append` maybe? otherwise the `errmsg` will be overwritten.

Reply:
but `errmsg` is string, so maybe use `+=` to concatenate error message(s)

Reply:
Yes, it would be overwritten but with latest data i.e, the last msg will contain all the missing dirs. So we are not missing the data here. I think it is correct. I tested it on console with an example as well.
```
>>> dirs = ['/home/a225', '/data/user/a225', '/data/scratch/a225']
>>> missing_dirs=[]
>>> msg={}
>>> for d in dirs:
...     if not Path(d).exists():
...         missing_dirs.append(d)
...         msg['errmsg'] = f"Error: missing dirs {missing_dirs}"
...
>>> print(msg['errmsg'])
Error: missing dirs ['/home/a225', '/data/user/a225', '/data/scratch/a225']
```